src/Microsoft.ML.Experimental/DataLoadSave/Database/DatabaseLoaderCatalog.cs

```diff
@@ -11,20 +11,31 @@ namespace Microsoft.ML
     /// </summary>
     public static class DatabaseLoaderCatalog
     {
-        /// <summary>
-        /// Create a database loader <see cref="DatabaseLoader"/>.
-        /// </summary>
+        /// <summary>Create a database loader <see cref="DatabaseLoader"/>.</summary>
         /// <param name="catalog">The <see cref="DataOperationsCatalog"/> catalog.</param>
         /// <param name="columns">Array of columns <see cref="DatabaseLoader.Column"/> defining the schema.</param>
         public static DatabaseLoader CreateDatabaseLoader(this DataOperationsCatalog catalog,
             params DatabaseLoader.Column[] columns)
         {
-            var options = new DatabaseLoader.Options
-            {
+            var options = new DatabaseLoader.Options {
```

Comment:
(nit) Allman style says braces should go on a new line. Reformatting this code goes against the style guideline.

Reply:
This was my IDE, will fix. The .editorconfig should be updated to appropriately preference this.

It is not uncommon for object initializers and lambdas to use K&R while the rest of the code uses allman.

Reply:
Fixed.